src/main/python/afp_cli/cliv2.py

```diff
@@ -71,8 +74,9 @@ def unprotected_main():
         log.DEBUG = True
     debug(arguments)
 
-    # parse the subcommand, only one will be active
-    subcommand = [s for s in SUBCOMMANDS if arguments[s]][0]
+    # parse the subcommand, use SIMPLE mode if no subcommand
+    subcommand = [s for s in SUBCOMMANDS if arguments[s]]
+    subcommand = subcommand[0] if subcommand else SIMPLE
```

Comment:
You are right, I'll make the first line plural.

On 16 March 2016 08:04:56 CET, Schlomo Schapiro <notifications@github.com> wrote:
>> @@ -71,8 +74,9 @@ def unprotected_main():
>>          log.DEBUG = True
>>      debug(arguments)
>>  
>> -    # parse the subcommand, only one will be active
>> -    subcommand = [s for s in SUBCOMMANDS if arguments[s]][0]
>> +    # parse the subcommand, use SIMPLE mode if no subcommand
>> +    subcommand = [s for s in SUBCOMMANDS if arguments[s]]
>> +    subcommand = subcommand[0] if subcommand else SIMPLE
>
>Is it really good Python to change the type of the variable from list
>to scalar?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ImmobilienScout24/afp-cli/pull/54/files/864351f20760101624627fe4e2f3f5bb5a893a63#r56288980

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.